opentera-webrtc-web-client/src/DataChannelClient.js

```diff
@@ -61,7 +61,7 @@ class DataChannelClient extends SignalingClient {
 
   _connectDataChannelEvents(id, dataChannel) {
     dataChannel.onmessage = event => {
-      this._onDataChannelMessage(id, this.getClientName(id), this.getClientData(id), event.data);
+      this._onDataChannelMessage(event.data);
```

Comment:
This would need to be changed also when assigning these callbacks, to make sure to remove the other parameters. Are they really never used? Would they never be used in the future? Is that not a standard WebRTC interface?

Reply:
I believe this must be kept, we need to know which client is sending data.

Reply:
Ok, I'll keep the parameters and disable the unused parameter warnings for the specific events in the front-end

Reply:
Yes, the parameters are useful for some usecase. For example, we could use the framework to create a chat app, so we need to know the source of the message.